packages/components/src/context-tag/context-tag.tsx

```diff
@@ -1,20 +1,23 @@
+import { cloneElement } from 'react'
+
 import { ChevronRightIcon, PlayIcon } from '@status-im/icons'
 import { Stack, styled } from '@tamagui/core'
 import { View } from 'react-native'
 
 import { Avatar } from '../avatar'
 import { Text } from '../text'
 
+import type { AvatarProps } from '../avatar'
 import type { TextProps } from '../text'
+import type { IconProps } from '@status-im/icons'
 
 type Props = {
-  children?: React.ReactNode
-  size?: 24 | 32
+  size?: 20 | 24 | 32
```

Comment:
> Context tags with size of 24 were already defined above, weren't they – https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=1336:34320&mode=design#541570513

Reply:
It's defined and it's weird, it has different paddings, different font-size. That's not how DS should work.

Reply:
![Screenshot 2023-08-31 at 10 54 09](https://github.com/status-im/status-web/assets/520927/6b238505-9404-47c0-9250-9efc945d9d19)

Reply:
> These context sizes are the same size as the ones above (24px), but the text size differs. We can have a bigger or smaller text depending on the context.
> ...
> Could you please share their usage where it differs?
> – https://www.figma.com/file/IBmFKgGL1B4GzqD8LQTw6n?node-id=1336:34320&mode=design#541570513

Asked design for clarification but added `textSize` prop for `size={24}` for now.